### PR TITLE
Allow customization of battery module tooltip

### PR DESCRIPTION
**Pull request by Markaos**
This commit allows users to change tooltip format using `tooltip-format[-status][-state]` parameters. It also adds `autoTooltip` (I couldn't come up with anything better) format argument containing the "default" tooltip text.

This shouldn't affect existing configs

### CODE REVIEW

File: src/modules/battery.cpp [hunk 1]
```diff
@@ -169,22 +169,37 @@ auto waybar::modules::Battery::update() -> void {
...
   // Transform to lowercase  and replace space with dash
   std::transform(status.begin(), status.end(), status.begin(), [](char ch) {
     return ch == ' ' ? '-' : std::tolower(ch);
   });
   auto format = format_;
   auto state = getState(capacity, true);
+  auto time_remaining_formatted = formatTimeRemaining(time_remaining);
+  if (tooltipEnabled()) {
+    std::string tooltip_text_default;
+    std::string tooltip_format = "{autoTooltip}";
```
Comment (Alexays) [inline src/modules/battery.cpp:182]: Not a big fan of the `autoTooltip` naming.
Maybe something like `timeTo`

**Review by Alexays — CHANGES_REQUESTED** [review]
Sorry for the delay, here's a review :)

**Comment by Markaos** [comment]
I agree, `timeTo` sounds like a reasonable name

**Comment by Alexays** [comment]
Thanks!

**Comment by Alexays** [comment]
Could you update github wiki? :)

**Comment by Markaos** [comment]
Sure, I'm on it

**Pull request merged**